src/main.cpp

```diff
@@ -3996,14 +3996,18 @@ std::string GetWarnings(const std::string& strFor)
     if (fLargeWorkForkFound)
     {
         nPriority = 2000;
-        strStatusBar = strRPC = _("Warning: The network does not appear to fully agree! Some miners appear to be experiencing issues.");
+        strStatusBar = strRPC = "Warning: The network does not appear to fully agree! Some miners appear to be experiencing issues.";
     }
     else if (fLargeWorkInvalidChainFound)
     {
         nPriority = 2000;
-        strStatusBar = strRPC = _("Warning: We do not appear to fully agree with our peers! You may need to upgrade, or other nodes may need to upgrade.");
+        strStatusBar = strRPC = "Warning: We do not appear to fully agree with our peers! You may need to upgrade, or other nodes may need to upgrade.";
     }
 
+    // try to i18n the string
+    if (translate)
+        strStatusBar = _(strStatusBar.c_str());
```

Comment:
Ah. I get you point.
The missing `_(<string>)` will prevent the python script from auto-extracting strings-that-require translation. Hmm... right. This is suboptimal.